index.html

```diff
@@ -300,30 +300,24 @@ <h1 class="partner_heading t-align-center mar-x-2">Partners</h1>
     </div>
     <div class="get_involved mar-x-5">
         <h1 class="head_text t-align-center mar-x-5">Get involved</h1>
-        <div class="ways-to-container d-flex flex-d-column f-align-center">
-            <div class="contribute t-align-center mar-x-5 padd-bottom-2">
+        <div class="ways-to-container d-flex flex-d-column f-align-center get_involved_wrapper">
+            <div class="contribute t-align-center mar-x-5 padd-bottom-2 box box-1">
                 <h3 class="padd-5 mar-bottom-4 ">Contribute code</h3>
```

Comment:
Try removing the class:  padd-5  from the h3 in the get involved section. @Evavic44

Reply:
Removed the ``pad-5`` class from each the ``h3`` headings. Please check it out.